backend/db/refresh.py

```diff
@@ -57,6 +59,7 @@ def refresh_db(
         if since and dp.parse(since) > dp.parse(last_refresh) and not force_non_contiguity:
             raise ValueError(SINCE_ERR)
         since = since if since else last_refresh
+        since = dp.parse(since) - timedelta(hours=buffer_hours)
```

Comment:
### PSA about API staiblity
I did actually get a `503 Service Unavailable Error` earlier today, FYI.

I also was doing a run and it was stuck for an hour in `requests.request()`. Worked fine when I restarted the process.